### PR TITLE
Add microshift action

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR adds support for Microshift action and tweaks the `verify.sh` accordingly. We can discuss using this PR's `verify.sh` changes everywhere where `verify.sh` is used.

Also, we might want the latest tag on `local-dev-cluster` repo. As of now, I have used `local_dev_cluster_version: main`

CC: @SamYuan1990, @rootfs

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -52,20 +50,42 @@ jobs:
         runningBranch: kind
         bcc_version: 0.25.0
         kubectl_version: 1.27.2
-        local_dev_cluster_version: v0.0.0
+        local_dev_cluster_version: main
```
Comment (rootfs) [inline .github/workflows/test.yml:53]: let's stay with tag or sha

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -52,20 +50,42 @@ jobs:
...
   test_microshift:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
     - uses: ./
       with:
-        runningBranch: microshift
\ No newline at end of file
+        runningBranch: microshift
+        local_dev_cluster_version: main
```
Comment (rootfs) [inline .github/workflows/test.yml:70]: tag or sha

File: .github/workflows/test.yml (hunk 1 above)
Comment (SamYuan1990) [reply .github/workflows/test.yml:53]: no sha, branch or tag.
here we use the main branch to keep the latest dev version.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -52,20 +50,42 @@ jobs:
         runningBranch: kind
```
Comment (SamYuan1990) [inline .github/workflows/test.yml:50]: we can use a metric for running branch.

**Review by SamYuan1990 — CHANGES_REQUESTED** [review]
Is there any possible to make it works by parameter ? follow don't copy paste our self.

**Comment by SamYuan1990** [comment]
btw, @vprashar2929 , please take a look at  https://github.com/sustainable-computing-io/local-dev-cluster/pull/20

File: .github/workflows/test.yml (hunk 1 above)
Comment (vprashar2929) [reply .github/workflows/test.yml:53]: I totally agree with you @SamYuan1990 on this. I gave this a thought and in some situations where there are some enhancements/major changes done to local_dev_cluster(which are experimental) and we don't want to support that change everywhere else then in that situation sha or tagging might help us to be on stable side. I might be wrong with this assumption. Wdyt?

File: index.js [hunk 4]
```diff
@@ -93,7 +93,48 @@ async function kind() {
...
+    if (parameterExport !=""){
+      parameterExport = parameterExport+" && "
+    }
+    parameterExport=parameterExport+"export GRAFANA_ENABLE="+grafana_enable;
+  }
+  if (parameterExport !=""){
+    parameterExport=parameterExport+" && "
+  }
+  core.debug(parameterExport);
+  shell.exec(parameterExport +" cd local-dev-cluster && bash -c './main.sh microshift_up'")
```
[the remark this answers is not in the document]
Comment (vprashar2929) [reply index.js:137]: Yes, the only difference between the two is exporting KIND_VERSION and calling `./main.sh` with a different arg. But we can have one function and make it parameterized. I will update this

File: .github/workflows/test.yml (hunk 1 above)
Comment (SamYuan1990) [reply .github/workflows/test.yml:61]: I suppose using main here is good enough for now. @rootfs wdyt?

**Review by rootfs — APPROVED** [review]

**Review by SamYuan1990 — APPROVED** [review]
I am going to take local holiday and I don't want to block this PR during local holiday.
Hence, I will give my approve base on today's code base.

**Pull request merged**